libraries/plugins/xfpga/usrclk/fpga_user_clk.c

```diff
@@ -151,10 +153,29 @@ struct dfh {
 			uint64_t type : 4;
 		};
 	};
+	char guid_l[64];
+	char guid_h[64];
```

Comment:
`fpga_guid` is compatible with libuuid's uuid_t, so users can use libuuid
 fpga_guid is defined in /include/opae/types.h
use fpga_guid varabile